src/genie/libs/parser/iosxe/show_cdp.py

```diff
@@ -248,7 +248,7 @@ def cli(self, output=None):
         # VTP Management Domain: 'Accounting Group'
```

Comment:
can you please an example of a string that would support this change?

Reply:
VTP Management Domain: '4074'

Reply:
I am sorry, I meant "please add an example". Just add a string on top of the existing one

Reply:
Hi domachad, I added Test-examples. Please let me know what else is needed. I am sorry but this is my first pull-request, so I kindly ask for your support here. thank you!